controllers/default.py

```diff
@@ -940,10 +940,6 @@ def mark_friend():
                                          "friend_requests",
                                          scheme=True,
                                          host=True,
-                                         extension=False),
```

Comment:
Oops... sorry, didn't know that is what you meant.
It also looks kinda not right to me hardcoding a url in code.
Thanks for the feedback and patience